mix.exs

```diff
@@ -13,7 +13,11 @@ defmodule ElixirBase.Mixfile do
       escript: [main_module: ElixirBase.CLI],
       docs: [extras: ["README.md"], output: "./doc/app"],
       deps: deps(),
-      aliases: aliases()
+      aliases: aliases(),
+      dialyzer: [
+        plt_add_apps: [:mix],
+        flags: [:unmatched_returns, :error_handling, :race_conditions, :no_opaque, :underspecs]
```

Comment:
I think it is okay to skip `overspecs` and `specdiffs` as they are not recommended to use by Dialyzer developers